### PR TITLE
feat(editor): added editor menu

### PR DESCRIPTION
**Pull request by makhnatkin**
We want to add a new section to Gravity-UI
[Demo here](https://landing.makhnatkin.ui.yandex-team.ru/markdown-editor)

1. In this PR, we had to downgrade the version of @diplodoc/transform to be compatible with the editor.
2. There is currently a problem with the sticky header editor, probably due to Next.js.

### CODE REVIEW

File: src/components/MarkdownEditor/MarkdownEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import {MarkdownEditorView, useMarkdownEditor, wToolbarConfig} from '@gravity-ui/markdown-editor';
+import {mToolbarConfig} from '@gravity-ui/markdown-editor/_/bundle/config/markup.js';
```
Comment (d3m1d0v) [inline src/components/MarkdownEditor/MarkdownEditor.tsx:2]: You can import `mToolbarConfig` from root export of `@gravity-ui/markdown-editor`

File: src/components/MarkdownEditor/MarkdownEditor.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import {Col, Grid, Row} from '@gravity-ui/page-constructor';
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply src/components/MarkdownEditor/MarkdownEditor.tsx:3]: lets fix it in next PR-s in:
```
src/components/MDXRenderer/MDXRenderer.tsx
src/components/ComponentsLibrary/ComponentsLibrary.tsx
src/components/DesignSection/DesignSection.tsx
src/components/Icons/Icons.tsx
src/components/MarkdownEditor/MarkdownEditor.tsx
src/components/Menu/Menu.tsx
src/components/Themes/Themes.tsx
```

File: src/components/MarkdownEditor/MarkdownEditor.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                        >
+                            {t('goToLibrary')}
+                        </Button>
+                    </div>
+                </Col>
+            </Row>
+            <Row>
+                <Col sizes={12}>
+                    {isClient && (
+                        <ThemeProvider theme="dark">
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply src/components/MarkdownEditor/MarkdownEditor.tsx:78]: I got error without it:
```
Unhandled Runtime Error
Error: useTheme* hooks must be used within ThemeProvider
```

File: src/components/MarkdownEditor/MarkdownEditor.tsx (hunk 1 above)
Comment (makhnatkin) [reply src/components/MarkdownEditor/MarkdownEditor.tsx:2]: thx `markupToolbarConfigs. mToolbarConfig`

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**